doc/JSON-RPC-interface.md

```diff
@@ -0,0 +1,105 @@
+JSON-RPC Interface
+==================
+
+The JSON-RPC API is by default enabled in bitcoind and disabled on bitcoin-qt.
+This can be changed with `-noserver` and `-server` options. In bitcoin-qt it is
+possible to issue execute RPC methods in the Debug Console Dialog.
+
+*Add some basic examples like:*
+
+*To see the available methods use `bitcoin-cli -regtest help`*
+
+#### Server Configuration
```

Comment:
Increment header levels one level at a time. You might want to use `mdl` to check the Markdown formatting :-)